backend/src/main/java/com/klm/dev/exercises/devcase02/randomquote/RandomQuoteClient.java

```diff
@@ -1,18 +1,67 @@
 package com.klm.dev.exercises.devcase02.randomquote;
 
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
 import org.springframework.http.*;
 import org.springframework.stereotype.Service;
 import org.springframework.web.client.RestTemplate;
+import org.springframework.stereotype.*;
+import org.springframework.beans.factory.annotation.*;
 
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.*;
 
 @Service
+@Configuration
+@PropertySource("classpath:RandomQuote.properties")
+
 public class RandomQuoteClient {
-   private RestTemplate restTemplate = new RestTemplate();
 
-   public Quote getQuote() {
-        return restTemplate.getForObject("http://gturnquist-quoters.cfapps.io/api/random", Quote.class);
+    private RestTemplate restTemplate = new RestTemplate();
+    private static final int NTHREDS = 10;
+
+    @Value("${quote.url}")
+    private String url;
+
+    @Value("${repeat}")
+    private int repeat;
+
+    public Quote getQuote() {
+        return restTemplate.getForObject(url, Quote.class);
     }
 
+    public List<Quote> getQuotes() {
+        ExecutorService executor = Executors.newFixedThreadPool(NTHREDS);
```

Comment:
try to limit the amount of tasks that can be added to 50, and increase the repeat for this to 200